robot/EDA/resources/locators_53.py

```diff
@@ -0,0 +1,414 @@
+""" Locators for Winter '22 """
```

Comment:
I think this needs to be something like:
```
from locators_54 import *
import copy
eda_lex_locators = copy.deepcopy(eda_lex_locators)
#Override the locator selector value
eda_lex_locators['eda_settings_new']['dropdown_input']="//label[text()='{}']/../descendant::input[@role='combobox']"
```

Reply:
Committed this change. Robot tests in meta ci are passing